examples/modules/comparison/plot_3_compare_sorter_with_ground_truth.py

```diff
@@ -51,14 +51,14 @@
 # This function first matches the ground-truth and spike sorted units, and
 # then it computes several performance metrics.
 # 
-# Once the spike trains are matched, each spike is labelled as: - true
-# positive (tp): spike found both in :code:`gt_sorting` and :code:`tested_sorting`
-# - false negative (fn): spike found in :code:`gt_sorting`, but not in
-# :code:`tested_sorting` - false positive (fp): spike found in
-# :code:`tested_sorting`, but not in :code:`gt_sorting` - misclassification errors
-# (cl): spike found in :code:`gt_sorting`, not in :code:`tested_sorting`, found in
-# another matched spike train of :code:`tested_sorting`, and not labelled as
-# true positives
+# Once the spike trains are matched, each spike is labelled as: 
+# 
+# - true positive (tp): spike found both in :code:`gt_sorting` and :code:`tested_sorting`
+# - false negative (fn): spike found in :code:`gt_sorting`, but not in :code:`tested_sorting` 
+# - false positive (fp): spike found in :code:`tested_sorting`, but not in :code:`gt_sorting` 
+# - misclassification errors (cl): spike found in :code:`gt_sorting`, not in 
```

Comment:
This is not more the case in actual version.
Misclassification errors are now FN.

Reply:
Got you. I'll close this PR.